internal/operator/client.go

```diff
@@ -82,7 +81,7 @@ func NewOpCapClient() (Client, error) {
 
 	client, err := runtimeClient.New(kubeConfig(), runtimeClient.Options{Scheme: scheme})
 	if err != nil {
-		logger.Errorf("could not get subscription client")
+		zap.S().Info("backend/local: apply calling Apply")
```

Comment:
Can we change this to `logger`or `log` instead of `zap.S()`? Same comment for all changes.

Reply:
I've followed this way to avoid redeclaring the zap Logger for each function. I've changed the main.go, calling the Zap as Global Sugared Logger.

Reply:
I've just applied the looger.